src/lib/modernExtend.ts

```diff
@@ -1646,6 +1685,28 @@ export function electricityMeter(args?: ElectricityMeterArgs): ModernExtend {
         toZigbee = [tz.electrical_measurement_power, tz.acvoltage, tz.accurrent];
         delete configureLookup.seMetering;
     }
+    if (args.threePhase !== true) {
```

Comment:
```suggestion
    if (args.threePhase) {
```

Reply:
Used the same pattern as configureReporting instead. As we need to check for both true and false values. It's been a while since i wrote js/ts.

Could you please review this once more?